docs/core-concepts/model-basics.md

```diff
@@ -278,6 +278,35 @@ Foo.init({ /* attributes */ }, {
 });
 ```
 
+## Prevent creating a default PK attribute
+
+By default, Sequelize automatically adds the primary key field `id` to every model when there is no primary key field defined. When there is an `id` field but not a primary key field Sequelize will throw an error. To prevent this you can use the option `noPrimaryKey` when creating the instance and set it to true.
+
+```js
+const sequelize = Sequelize.createSequelizeInstance({
+  define: {
+    noPrimaryKey: true,
+  },
+});
+
+const User = sequelize.define('User', {
+  name: {
+    type: DataTypes.STRING
+  }
+});
+```
+
+In case you only want to disable primary keys on a single model, you can define the field with `primaryKey: false`.
```

Comment:
That's not true. Doing that will still add a primary key field called 'id'. It's only going to make `name` not a primary key, which would already be the case without the option.

This is how you disable adding the primary key using local options:

```js
const User = sequelize.define('User', {
  name: DataTypes.STRING,
}, {
  noPrimaryKey: true,
});
```

I would also suggest switching the order of explanations to start with the local options first, then the global ones (as we always do throughout the documentation)